doc/cephfs/experimental-features.rst

```diff
@@ -77,5 +84,8 @@ will not have been experienced by anybody else -- knowledgeable help will be
 extremely limited. You also probably do not have the security or isolation
 guarantees you want or think you have upon doing so.
 
+Note that snapshots and multiple filesystems are *not* tested and probably
```

Comment:
Let's not under-sell ourselves, they are tested to an extent.  Maybe we should just fall back to our catch-all language and say they're experimental?